backend/tarsy/agents/prompts/builders.py

```
@@ -386,9 +386,9 @@ def parse_react_response(self, response: str) -> Dict[str, Any]:
     def get_react_continuation_prompt(self, context_type: str = "general") -> List[str]:
         """Get ReAct continuation prompts for when LLM provides incomplete responses."""
         prompts = {
-            "general": "Observation: Choose ONE option: (1) Continue investigating with 'Thought: [reasoning] Action: [tool] Action Input: [params]' then STOP (do NOT generate fake observations) OR (2) Conclude with 'Thought: I have sufficient information Final Answer: [your analysis]'",
-            "data_collection": "Observation: Choose ONE option: (1) Continue data collection with 'Thought: [reasoning] Action: [tool] Action Input: [params]' then STOP (do NOT generate fake observations) OR (2) Conclude with 'Thought: I have sufficient data Final Answer: [data summary]'",
-            "analysis": "Observation: Choose ONE option: (1) Continue investigating with 'Thought: [reasoning] Action: [tool] Action Input: [params]' then STOP (do NOT generate fake observations) OR (2) Conclude with 'Thought: I have sufficient information Final Answer: [complete analysis]'"
+            "general": "Observation: 🔥 REMEMBER: Use 'Thought:' (with colon). Choose ONE option: (1) Continue investigating with 'Thought: [reasoning] Action: [tool] Action Input: [params]' then STOP (do NOT generate fake observations) OR (2) Conclude with 'Thought: I have sufficient information Final Answer: [your analysis]'",
```

Ruff (0.12.2)

Line too long (330 > 88)

(E501)